### DISCUSSION TITLE
Possible to use "modelessly", ie. without Emacs-mode / Vim-mode shortcut acrobatics?

### DISCUSSION THREAD

**Discussion by metaleap** [lem-project/lem#1391] [Q&A]
Lem looks promising to me because it doesn't look as abandoned as most other editor plugins for Lisp (apart from those for Emacs / Vim, both of which I really don't enjoy).

But I'm wondering whether it'll be fully customizable towards contemporary (or simply, post-80s) keyboard shortcuts, such as you'd find in a Sublime / VSCode / Geany and practically *all* others *except* Emacs / Vim...

The next question would be if someone already has a ready import script or config file for such  =)

(Btw. that doesn't mean I'm "a mouse user" of my editors, I've always keyboard-driven them, I just like them to behave like any other editable controls in the last 20+ years, with ctrl+x / ctrl+c / ctrl+v for clipboard, ctrl+a to select all ... and select with shift + arrows, hop words with ctrl + arrows, tabbing through UX, ctrl+tab to flip through buffer, alt for menus, and so on.)

**Comment by vindarel** [comment]
> all others except Emacs / Vim

and the Linux terminal :p (C-a, C-e, C-r, M-/…) (kidding but yeah)

Lem is very much moldable, so it will be possible to bind the keys to other actions. I don't know of a ready-to-use config for this but I'm sure some people have something to help (you can also ask on Discord).

paste is `yank` so you can do, for a start:

```
(in-package :lem-user)
(define-key *global-keymap* "C-v" 'yank)
(define-key *global-keymap* "C-c" 'copy-region)
```

C-x is a prefix for other commands so that could be annoying.

You can also… take some time and learn the bindings…
